app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -603,6 +604,19 @@ public void onStopTrackingTouch(SeekBar seekBar) {
         }
     }
 
+    private static String getWebsiteLinkWithFallback(Playable media) {
```

Comment:
In my opinion, this function belongs to `FeedItemUtil`

Reply:
The bulk of the logic is already in `FeedItemUtil#getLinkWithFallback()`

Here, in `MediaplayerActivity#getWebsiteLinkWithFallback()`, the input is the more generic `Playable`, rather than `FeedItem`. It basically fallback to `FeedItemUtil#getLinkWithFallback()` if the `Playable` is a `FeedItem`.